docs/releasing.md

```diff
@@ -1,7 +1,7 @@
 # Releasing New Versions of rok8s-scripts
 
 ## Before Merging the Pull Request!
-Please be sure to update `./setup.py` and `./package.json` to reflect the new version you wish to release. CODEOWNERS should be able to answer what the next version to release should be.
+Please be sure to update `./package.json` to reflect the new version you wish to release. CODEOWNERS should be able to answer what the next version to release should be.
```

Comment:
package.json has also been removed here

Reply:
I just removed this section.  I don't believe there is anywhere else that needs version updating.